Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,221 @@
+                inc=incidentDetail(str(incident1))
+                serialized_obj = helpers.serialize_object(inc[0])
+                rawjson = json.dumps(serialized_obj,default = myconverter)
+                incidents.append({
+                    'name':"DLP Incident #"+str(incident1),
+                    'details':  inc[0]['incident']['violatedPolicyRule'][0]['ruleName']+ "RULE has been violated"
+                    'rawJSON': rawjson,
+                    'severity' : SEVERITY[inc[0]['incident']['severity']]
+                })
+                if inc[0]['incident']['incidentCreationDate'].isoformat() > last_run['lt_time'].isoformat():
+                    demisto.setLastRun({'lt_time':inc[0]['incident']['incidentCreationDate'].strftime("%d/%m/%Y %H:%M:%S.%f")})
+        demisto.incidents(incidents)
+
+
+    def updateIncidents():
+        args = demisto.args()
+        inc_id = args['incidentId']
+        args.pop('incidentId')
+        if args['status'] == 'New':
+            args['status'] = 'incident.status.New'
+        return client.service.updateIncidents(updateBatch={'batchId':str(random.randint(11111,99999)),'incidentId':inc_id,'incidentAttributes':args})
+
+
+
+    if demisto.command() == 'fetch-incidents':
+        fetchincidents()
+        
+    elif demisto.command() =='test-module':
+        try:
+            inc_list = get_incident_id(datetime.datetime.now())
+            demisto.results("ok")
+        except:
+            return_error("Invalid instance details")
+            
+    elif demisto.command() == 'dlp-incident-details':
+        inc_details = incidentDetail( demisto.args()['incidentId'])
+        res = helpers.serialize_object(inc_details[0])
+        json_res = json.loads(json.dumps(res,default = myconverter))
+        demisto.results({'ContentsFormat': formats['json'],
+        'Type': entryTypes['note'],
+        'Contents':json_res,
+        'EntryContext':{'Symantec':{"DLP":{'Incident':json_res}}}
+        })
+        
+    elif demisto.command() == 'dlp-update-incident':
+        demisto.results(updateIncidents())
+
+  type: python
+  commands:
+  - name: dlp-incident-details
+    arguments:
+    - name: incidentId
+      required: true
+      description: incident Id  from DLP console to fetch the data of
+      outputs:
+    - contextPath: Symantec.DLP.Incident
+      description: DLP incident details
+      type: unknown
+    description: fetches DLP incident details
+  - name: dlp-update-incident
+    arguments:
+    - name: incidentId
+      required: true
+      description: incident Id  from DLP console to fetch the data of
+    - name: severity
+      auto: PREDEFINED
+      predefined:
+      - HIGH
+      - MEDIUM
+      - LOW
+      - INFO
+      description: changes the severity of the Incident
+    - name: status
+      auto: PREDEFINED
+      predefined:
+      - New
+      - Escalated
+      - Investigation
+      - Resolved
+      - Dissmissed
```

Comment:
@ssudheerk99 changing the status to Resolved or Dissmissed will close the incident in DLP?

Reply:
yes, but both are used in different ways
Dismissed is used when an alert is received and no action is to be taken and can be closed directly.
Resolved is used when an alert is received and some action has to be taken like escalating to manager or some thing else and the incident will be marked as resolved when all the process is completed.